www/branch.js

```diff
@@ -263,65 +270,51 @@ Branch.prototype.createBranchUniversalObject = function (options) {
              *    |  $blackberry_url   |   String   |   Blackberry URL  |
              *    | $windows_phone_url |   String   |  Kindle Fire URL  |
              *    -------------------------------------------------------
-             */
```

Comment:
The reason why the callbacks should be set upon calling the `showShareSheet` is that it wouldn't be able to catch/listen to the event on the first trigger (but it works on the second and so on).